activity_hub.php

```diff
@@ -128,7 +128,7 @@
 
 // Proofreaders with fewer than 21 pages can't see the filter box on the Round
 // pages so prevent those users from selecting the filtered option.
-if ($pagesproofed <= 20) {
+if ($pagesproofed < 20) {
```

Comment:
These constants (20, 100, 300) sprinkled throughout multiple files makes me want to ask if we can put these in a shared location to make updating them in the future easier?

Reply:
I'm hoping to resolve that more holistically via #995 -- right now there's not a lot of cohesion and I can't come up with constant names that actually describe them as they are all very bespoke. This PR is decidedly only a bandaid.